modules/i18n/src/translations/en-US/portals/applications.ts

```diff
@@ -1427,7 +1434,9 @@ export const applications: ApplicationsNS = {
                 }
             },
             managementAppBanner: "The application is allowed to access the management APIs of this " +
-                "organization."
+                "organization.",
+            discoverableGroupsBanner: "Since you haven't selected any discoverable groups, the application will" +
+                " be visible to everyone in the organization."
```

Comment:
WDYT?

```suggestion
            discoverableGroupsBanner: "Having no discoverable groups configured will make the application visible to everyone in the organization from the My Account portal."
```

cc: @himeshsiriwardana

Reply:
Ack, will improve the error message

Reply:
The warning message was removed, and radio options were introduced for an improved user experience.

Reply:
Just thinking whether we can eliminate the discoverable checkbox with a single radio field like this.

<img width="324" alt="Screenshot 2025-02-26 at 08 22 44" src="https://github.com/user-attachments/assets/a81040c8-7b23-4675-bc3f-e06fb490a4e9" />